packages/core/scripts/local/getGasEthHistoricalPrice.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @notice This is an example script demonstrating how to get a historical median gas price at a specific timestamp.
+ * It could be used for example to query a GASETH-TWAP-1Mx1M price before committing a vote for a DVM price request.
+ *
+ * @dev Prequisites: before running, you will need to set a GOOGLE_APPLICATIONS_CREDENTIALS environment variable for a service account.
+ * @dev This service account will need GCP admin or BigQuery permissions. This guide provides further instructions: https://cloud.google.com/docs/authentication/getting-started
+ * @dev How to run: yarn truffle exec ./scripts/local/getGasEthHistoricalPrice.js --network mainnet_mnemonic --time 1612137600
```

Comment:
nit: should the example contain `identifier`?

Reply:
Hm yeah probably. Was initially thinking this was only usable for one identifier, but it would also work for the `uGAS-FEB21` and `uGAS-MAR21` identifiers, so I will include.